GetIntoTeachingApi/Jobs/LocationBatchJob.cs

```diff
@@ -56,7 +56,7 @@ public async Task RunAsync(string batchJson)
 
         private static Location CreateLocation(dynamic location)
         {
-            return new Location(location.Postcode, location.Latitude, location.Longitude);
+            return new Location(location.Postcode, location.Latitude, location.Longitude, Source.CSV);

```

Comment:
Can we cover the source with a test in `LocationBatchJobTests` by putting something in here:

https://github.com/DFE-Digital/get-into-teaching-api/blob/master/GetIntoTeachingApiTests/Jobs/LocationBatchJobTests.cs#L49

Along the lines of:

```
var locations = DbContext.Locations.ToList();
locations.Count().Should().Be(batch.Count());
locations.All(l =>batch.Any(b => BatchLocationMatchesExistingLocation(b, l))).Should().BeTrue();
locations.All(l => l.Source == Soruce.Csv)
```